addons/tmi/services/pronouns.gd

```diff
@@ -19,19 +19,25 @@ func enrich(obj: TmiAsyncState):
 	if "pronouns" in profile.extra:
 		return
 	
-	var result = await utils.fetch(self, "https://pronouns.alejo.io/api/users/%s" % profile.display_name, HTTPClient.METHOD_GET, {}, {}, true)
+	var result = await utils.fetch(self, "https://api.pronouns.alejo.io/v1/users/%s" % profile.display_name, HTTPClient.METHOD_GET, {}, {}, true)
 	
 	if result.code != 200:
 		return
 	
 	if result.data.is_empty():
 		return
 	
-	var user_pronoun = result.data.front()
+	var user_pronoun = result.data
+	var primary_pronoun_id
+	var secondary_pronoun_id
 	if user_pronoun:
-		user_pronoun = user_pronoun.pronoun_id
+		primary_pronoun_id = user_pronoun.pronoun_id
+		secondary_pronoun_id = user_pronoun.alt_pronoun_id
+		var pronoun = _pronouns[primary_pronoun_id]
+		profile.extra["pronouns"] = pronoun.subject + "/"
 		
-	if user_pronoun:
-		var pronoun = _pronouns.filter(func (p): return p.name == user_pronoun).front()
-		profile.extra["pronouns"] = pronoun.display
-
+		if secondary_pronoun_id != null:
+			var secondary_pronoun = _pronouns[secondary_pronoun_id]
+			profile.extra["pronouns"] += secondary_pronoun.subject
+		else:
+			profile.extra["pronouns"] += pronoun.object
```

Comment:
Since the pronoun response and id mapping are both dictionaries instead of array now, this can be simplified to a few less steps.

```suggestion
	var user_pronoun = result.data
	var primary = _pronouns.get(user_pronoun.pronoun_id)
	var secondary = _pronouns.get(user_pronoun.alt_pronoun_id, primary)
	profile.extra["pronouns"] = "%s/%s" % [
		primary.subject,
		secondary.subject if secondary != primary else secondary.object
	]
```